next/pages/get-jwt.tsx

```diff
@@ -10,17 +11,36 @@ import { Wrapper } from 'components/styleguide/Wrapper'
 import { GetServerSidePropsContext } from 'next'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
 import { useState } from 'react'
-
 import { useEffectOnce } from 'usehooks-ts'
 
 const CognitoPrototype = ({ page }: AsyncServerProps<typeof getServerSideProps>) => {
-  const { getAccessToken } = useAccount()
+  const { getAccessToken, isAuth, updateUserData } = useAccount()
   const [accessToken, setAccessToken] = useState('')
 
-  useEffectOnce(() => {
-    getAccessToken().then(setAccessToken).catch(console.log)
+  useEffectOnce(async () => {
+    try {
+      const token = await getAccessToken()
+      if (token) {
+        setAccessToken(token)
+        const user = await getUserApi(token)
+        console.log(user)
+      }
+    } catch (error) {
+      console.log(error)
+    }
   })
 
+  const resetRc = async () => {
+    try {
+      await resetRcApi(accessToken)
+      const res = await updateUserData({ tier: null })
```

Comment:
whoa, this can be done ? this resets the data in cognito or just locally ?

would expect this not to be possible, and the approach here being reloading the user data from cognito after resetRcApi

Reply:
I've set write permission to tier attribute in Cognito console. Just for the test. 
Will not be possible in production.